sem_covid/services/enrich_pipelines/eu_cellar_enrich_pipeline.py

```diff
@@ -0,0 +1,42 @@
+import numpy as np
+import pandas as pd
+
+from sem_covid import config
+from sem_covid.services.enrich_pipelines.base_enrich_pipeline import BaseEnrichPipeline, BasePrepareDatasetPipeline, \
+    EMBEDDING_COLUMN
+from sem_covid.services.sc_wrangling.mean_vectorizer import text_to_vector
+
+EU_CELLAR_TEXT_COLUMNS = ['title']
+
+
+class EuCellarPreparePipeline(BasePrepareDatasetPipeline):
+
+    def __init__(self):
```

Comment:
this most likely will NOT work. Do not aim at instantiating a pipeline class and then pass methods as step operators in an Airflow DAG. 

They shall me static/class methods.